Makefile

```diff
@@ -0,0 +1,30 @@
+
+DOCKER ?= $(shell which docker)
+DOC_FILES := \
+	README.md \
+	code-of-conduct.md \
+	ROADMAP.md \
+	implementations.md \
+	bundle.md \
+	runtime.md \
+	runtime-linux.md \
+	config.md \
+	config-linux.md \
+	runtime-config.md \
+	runtime-config-linux.md
```

Comment:
On Wed, Dec 09, 2015 at 09:37:07AM -0800, Tianon Gravi wrote:
> > +DOC_FILES := \
> > +	README.md \
> > +	code-of-conduct.md \
> > +	ROADMAP.md \
> > +	implementations.md \
> > +	bundle.md \
> > +	runtime.md \
> > +	runtime-linux.md \
> > +	config.md \
> > +	config-linux.md \
> > +	runtime-config.md \
> > +	runtime-config-linux.md

List is missing principles.md (insert after code-of-conduct.md?).